docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+
+= Configuring a run configuration
+
+You can configure a number of separate run configurations to run against each of your projects. Each configuration is saved automatically as you work.
+
+.Prerequisites
+
+* Install `mta-cli`.
```

Comment:
No. We discussed this. mta-cli needs to be installed but it is not a prerequisite. You need to add mta-cli installation to the extension installation instructions because it is triggered within the UI when you install. It does not belong in prerequisites if the installation forces you to install it. Remove this from prerequisites